### PR TITLE
Docker images for SAS

### PR DESCRIPTION
**Pull request by seongsujeong**
This PR updates lock file to be used for generating Docker image to be delivered in cal/val release.

### CODE REVIEW

File: Docker/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FROM oraclelinux:8
 
 LABEL author="OPERA ADT" \
-    description="RTC gamma release R3" \
-    version="0.4-calval"
+    description="RTC gamma release R4" \
+    version="0.4-gamma"
```
Comment (gshiroma) [inline Docker/Dockerfile:5]:
```suggestion
    description="RTC cal/val release R4" \
    version="0.4-calval"
```

**Review by gshiroma — APPROVED** [review]
LGTM, @seongsujeong . Thank you. I just have one minor comment below.

File: Docker/Dockerfile (hunk 1 above)
Comment (seongsujeong) [reply Docker/Dockerfile:5]: Thanks @gshiroma. I was about to ask you about the label ;)

**Comment by seongsujeong** [comment]
It looks like the tests using lock file has passed, but on-the-fly dependency solution using environment.yaml needs to be revised.
That means the RTC-S1 SAS is good to release, but we will need to update `environment.yaml` as time permits.

**Pull request merged**